docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -136,4 +136,34 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 | --- | --- | --- | --- | --- |
 | tiny_yolov2 | Yes | Yes | Yes | Yes* |
 
-*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
\ No newline at end of file
+*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
+
+## CSharp API
+
+To use csharp api for openvino execution provider create a custom nuget package.
+
+1. Windows
+
+Build a custom nuget package for windows.
+```
+.\build.bat --config Debug --build --use_openvino $Device --build_csharp
+msbuild csharp\OnnxRuntime.CSharp.proj /p:OrtPackageId=Microsoft.ML.OnnxRuntime.Openvino /p:Configuration=Debug /t:CreatePackage
+```
+
+2. Linux
+
+We currently do not have a process to build directly in Linux. But we can
+copy shared library libonnxruntime.so to onnxruntime repository in windows
+and execute the same commands above to get custom nuget package for linux 
+
+On Linux Machine
+```
+./build.sh --config Debug --build_shared_lib --use_openvino $Device 
+```
+
+On Windows Machine
+```
+cp libonnxruntime.so $PATH/onnxruntime/ 
+.\build.bat --config Debug --build --use_openvino $Device --build_csharp
```

Comment:
What value should $PATH have? There are a few things it could be so a description would be better.

Reply:
Not sure this is clear enough for a new user. 

Where is libonnxruntime.so copied from? Assuming the build output directory from the Linux build, but there's no mention of where to find that like `<ORT linux repo>/build/Linux/<config>`.

Also feels a little hacky to just dump the linux .so file in the root directory of the ORT repo. Would be nicer to put it somewhere closer to where the C# build output would go, like `<ORT Windows repo>\csharp\src\Microsoft.ML.OnnxRuntime\bin\<config>\linux`.